.github/workflows/github-actions-test-boost-version.yml

```diff
@@ -0,0 +1,54 @@
+name: Test Boost Version
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      boostVersion:
+        description: 'Boost Version'
+        required: true
+        default: '1.81.0' 
```

Comment:
Can we have the dispatch receive a single version, but when it is triggered by the push or PR triggers then we run on multiple versions? If this cannot be one GHA that is fine, but it would be great if they were in the same.